hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/TransactionInfo.java

```diff
@@ -54,22 +55,25 @@ public record TransactionInfo(
         @Nullable AccountID payerID,
         @NonNull SignatureMap signatureMap,
         @NonNull Bytes signedBytes,
-        @NonNull HederaFunctionality functionality) {
+        @NonNull HederaFunctionality functionality,
+        @Nullable Bytes serializedTransaction) {
```

Comment:
Why is `serializedTransaction` necessary? It's easy to compute: 
`Transaction.PROTOBUF.toBytes(transaction)`

It'd be nice if we didn't have to add `, null` all over the place with this extra parameter

Reply:
It's quite expensive to re-serialize PBJ objects and since in the `EventTransaction` block item we need the serialized transaction, decided to just pass it along the whole way here. 🤷

Reply:
I forgot the serialization was so expensive 👍